pom.xml

```diff
@@ -58,7 +58,7 @@
     </parent>
 
     <properties>
-        <com.capitalone.dashboard.core.version>3.1.14</com.capitalone.dashboard.core.version>
+        <com.capitalone.dashboard.core.version>3.1.16-SNAPSHOT</com.capitalone.dashboard.core.version>
         <apache.rat.plugin.version>0.13</apache.rat.plugin.version>
```

Comment:
Do not push in code depending on a SNAPSHOT version of core. Wait for Ragha to create 3.1.16 (or a later) release.

Reply:
ok Ben